File: Java/1094.java
import java.util.Scanner;
import java.util.StringTokenizer;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        
        String C = sc.nextLine();
        int count = Integer.parseInt(C);
        
        String number = sc.nextLine();
        
        StringTokenizer token = new StringTokenizer(number, " ");
        
        int arr[] = new int[count];
        
        int i = 0;
        
        while(token.hasMoreTokens()) {
	    	arr[i] = Integer.parseInt(token.nextToken());
	    	i++;
        }
        
        for(int j = count - 1; j >= 0; j--) {
        	System.out.printf("%d ", arr[j]);
        }
    }
}

File: C++/1094.cpp
#include <iostream>

using namespace std;

int main(){
	int count;
	cin >> count;
	
	int arr[10000];
	
	int number;
	
	for(int i = 0; i < count; i++){
		cin >> arr[i];
	}
	
	for(int j = count; j > 0; j--){
		cout << arr[j-1] << " ";
	}
}